patroni/postgresql.py

```diff
@@ -105,6 +106,13 @@ def can_rewind(self):
                 data.get('Data page checksum version', '0') != '0'
         return False
 
+    @property
+    def sysid(self):
+        if not self._sysid:
+            data = self.controldata()
```

Comment:
self.controldata() returns empty dictionary under python3
this happens because subprocess.check_output returns bytestring instead of normal string.
solution - replace line #403 with:
data = data.decode().splitlines()  # yeah, decode is very important. it will covert bytestring into string